GDevelop.js/Bindings/Bindings.idl

```diff
@@ -780,10 +780,15 @@ interface BehaviorDefaultFlagClearer {
         [Ref] SerializerElement serializerElement);
 };
 
+interface UniquePtrBehavior {
+    Behavior get();
+    Behavior release();
+};
+
 interface Behavior {
     void Behavior();
 
-    Behavior Clone();
+    [Value] UniquePtrBehavior Clone();
```

Comment:
I see a change here but no change in any JS file that would use it.

Can you try remove it and see if it's actually used? If not, good let's not expose something like it to the JS.
If it's used, it means it probably need to call "release" (or "get").